Examples/CodePushDemoApp/android/app/proguard-rules.pro

```diff
@@ -15,3 +15,46 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
```

Comment:
Just a note that this file change is auto-generated from running the `react-native upgrade` command.